File: homework3/fizzBuzzExp.c
#include <stdio.h>

int main() {
    for (int i = 1; i <= 100; i++) {
        printf("%s%s%s\n", 
            i % 3 == 0 ? "Fizz" : "",
            i % 5 == 0 ? "Buzz" : "",
            (i % 3 != 0 && i % 5 != 0) ? printf("%d", i), "" : ""
        );
    }
    return 0;
}

File: homework3/FizzBuzzExp.java
public class FizzBuzzExp {
    public static void main(String[] args) {
        for (int i = 1; i <= 100; i++) {
            System.out.println(
                (i % 3 == 0 ? "Fizz" : "") +
                (i % 5 == 0 ? "Buzz" : "") +
                (i % 3 != 0 && i % 5 != 0 ? Integer.toString(i) : "")
            );
        }
    }
}